frontend/src/metabase/visualizations/components/Visualization/Visualization.unit.spec.js

```diff
@@ -21,6 +24,47 @@ describe("Visualization", () => {
     return container.querySelectorAll(`path[fill="${color}"]`);
   };
 
+  describe("with an error", () => {
+    // https://github.com/metabase/metabase/issues/49348
+    it("should render the error message and the proper title", async () => {
```

Comment:
For consistency, it would be nice to add the reference to the issue right in the test name instead of a comment above like `should render the error message and the proper title (metabase#49348)`